packages/engine/src/simulation/comms/message.rs

```diff
@@ -21,20 +23,23 @@ pub struct WrappedTask {
 
 #[derive(Debug)]
 pub struct EngineToWorkerPoolMsg {
+    pub span: Span,
```

Comment:
Should we make this guarded by a feature flag?

Reply:
Good question, have answered a similar discussion above so lets wait for resolution on that